api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -364,6 +364,15 @@ type FelixConfigurationSpec struct {
 	// [Default: Off].
 	// +optional
 	BPFLogLevel string `json:"bpfLogLevel" validate:"omitempty,bpfLogLevel"`
+	// BPFLogFilters is a comma-separated list of key=values where the value is
```

Comment:
The comment here needs to be updated to match the new `map[string]string` format.